File: javascript/quote.js
const Discord = require("discord.js");
const fetch = require("node-fetch")

const client = new Discord.Client();

function getQuote() {
  return fetch("https://zenquotes.io/api/random")
  .then(res => {
    return res.json()
  })
  .then(data => {
    return data[0]["q"] + "\n-" + data[0]["a"]
  })
}

client.on("ready", ()=> {
  console.log(`logged in as ${client.user.tag}`);
});

client.on("message", msg => {
  if(msg.author.bot) return
if(msg.content === "$quote"){
    getQuote()
    .then(quote => msg.channel.send(quote))
  }

});

client.login(process.env.TOKEN);

File: python/quote.py
import os
import discord
from dotenv import load_dotenv

load_dotenv()
token = os.getenv('PYTHON')

intents = discord.Intents.default()
intents.message_content = True

client = discord.Client(intents=intents)

@client.event
async def on_ready():
    print(f"logged in as {client.user}")

@client.event
async def on_message(message):
    if message.author == client.user:
        return
    
    if message.content.startswith("$hello"):
        await message.channel.send("Ahoy hoy")

client.run(token)